domain-classes-generator/src/main/scala/flatgraph/codegen/DomainClassesGenerator.scala

```diff
@@ -1196,16 +1157,70 @@ class DomainClassesGenerator(schema: Schema) {
         propertyKeySource(property, propertyKind = kindContexts.propertyKindByProperty(property))
       }
     }.mkString("\n\n")
-    val file = outputDir / "Properties.scala"
+    val propertiesFile = outputDir / "Properties.scala"
     os.write(
-      file,
+      propertiesFile,
       s"""package ${schema.basePackage}
          |
          |object Properties {
          |$propertyKeysConstantsSource
          |}""".stripMargin
     )
-    results.addOne(file)
+    results.addOne(propertiesFile)
+    val propertyNamesSource = schema.properties
+      .map { property =>
```

Comment:
did scalafmt format it like that? please confirm, I'd prefer to have the `.map` on the same line...

Reply:
It is indeed a scalafmt change